src/coreclr/tools/util/consoleargs.cpp

```diff
@@ -953,6 +953,11 @@ void ConsoleArgs::ProcessResponseArgs()
 #endif
 
         TextToArgs(szActualText, &listCurArg->next);
+
+        delete[] pwzFileBuffer;
+#ifndef TARGET_UNIX
+        delete[] szExpandedBuffer;
```

Comment:
does this need a null check before deletion?

Reply:
passing NULL to delete is valid

Reply:
See http://www.cplusplus.com/reference/new/operator%20delete/:
> Parameters
ptr
A pointer to the memory block to be released, type-casted to a void*.
If this is a null-pointer, the function does nothing.